source/client.c

```diff
@@ -2448,6 +2503,14 @@ static enum aws_mqtt_client_request_state s_unsubscribe_send(
         if (aws_channel_slot_send_message(task_arg->connection->slot, message, AWS_CHANNEL_DIR_WRITE)) {
             goto handle_error;
         }
+
+        /* TODO: timing should start from the message written into the socket, which is aws_io_message->on_completion
```

Comment:
Enter a ticket to follow up on this, or leave it out, or detail what the issues are. Otherwise, the next person to read this will have no idea what this is about.

Reply:
I am not sure about what the exact issue is. Mike brought it up [here](https://github.com/awslabs/aws-c-mqtt/pull/179#discussion_r592640136), will update the comments to be more clear.